api/test/factories.ts

```diff
@@ -352,7 +355,7 @@ export const generateFormationData = (
   legalStructureId?: FormationLegalType
 ): FormationData => {
   return {
-    formationFormData: generateFormationFormData({}, legalStructureId),
+    formationFormData: generateFormationFormData({}, legalStructureId, {}),
```

Comment:
this doesn't look good to read. As I read this line, my brain is thinking: "What is that empty object doing in this parameter list, what does it represent?" It's a lot like having a plain true/false in a parameter list. I think that if it's gonna be there, we should have named parameters so that this says `options: {}` or maybe this options thing should just optional, actually? even better. that way it's easy to read when it IS being used, but it's not causing questions when it is empty